data/avro-schema/src/main/resources/avro/net/corda/data/crypto/ShortHashes.avsc

```diff
@@ -0,0 +1,15 @@
+{
+  "type": "record",
+  "name": "ShortHashes",
+  "doc": "List of short hashes. A short hash is the first 12 characters of a hex string",
```

Comment:
better to say something like "List of short hashes. A short hash is the first 12 characters of the hex form of the hash. Short hashes should only be used when conciseness is important."

Reply:
actually, now I've looked at the implementation and commented there, better to simply allow unique prefixes, in which case either:

a) no AVRO change at all - anywhere you can specify a hash you can use a unique prefix
b) we have a more general AVRO change which allows a unique hash prefix to be sent, in a way that can be differentiated from a full hash.

Reply:
Doc updated